src/stories/tasks/confirmDelete/index.stories.mdx

```diff
@@ -0,0 +1,21 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import LinkTo from '@storybook/addon-links/react';
+
+<Meta title="Tasks/ConfirmDelete/Docs" />
+
+# ConfirmDelete
+
+**Confirms before deleting or refreshing an org that has unretrieved changes.**
+
+## Overview
+
+The **ConfirmDelete** component is an
+[SLDS Modal](https://react.lightningdesignsystem.com/components/modals/) used to
```

Comment:
TIL/took time to research the intricacies of the "a"/"an" rule. I knew "an SLDS component" sounds better than "a SLDS component" but never really thought about why.  The "an" is used before words that have a starting sound like a vowel... so in this case  the `S` in SLDS sounds like `ess` like `essential`. I know the whole "an" before vowels thing, or some words that start with h, like hour, but I never thought about other cases. Probably something I dozed off during back in 5th grade.